src/start/RealmScreen.js

```diff
@@ -53,6 +54,26 @@ class RealmScreen extends PureComponent<Props, State> {
 
     const { dispatch } = this.props;
 
+    let parsedRealm;
+    try {
+      parsedRealm = new WhatwgURL(realm);
+    } catch (err) {
```

Comment:
This bit can be simplified with `tryParseUrl` (which I added last week for a very similar pattern in #4034 :slightly_smiling_face: )